jenkins_jobs/sample_test_jobs/docker/sample_tests_beryllium/sample_tests_beryllium.yaml

```diff
@@ -0,0 +1,412 @@
+# Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.

```

Comment:
Comments for all files below: 
- why do we need separate conf files for docker-based tests? can't we just reuse the sample test confs we already have?
- let's not increase the complexity with adding support for previous test confs (Lithium, etc.)
- let's create sample Docker jobs only for 1 ODL release (Be)